src/control/lib/netdetect/README.md

```diff
@@ -0,0 +1,13 @@
+# Netdetect library
```

Comment:
It wasn't on my radar that we wanted such a section for the netdetect library.   I am not entirely clear on the goal of the section (for example, who the target audience is and then what we want them to really know).